mars/services/core.py

```diff
@@ -134,7 +134,7 @@ def _normalize_modules(modules: _ModulesType):
         modules = [modules]
     else:
         modules = list(modules)
-    modules = ["mars.services"] + modules
+    modules = [__name__.rsplit('.', 1)[0]] + modules
```

Comment:
Would it be better to add some comments? Telling the reviewers that this is for the integration with `xorbits`.

Reply:
OK. LGTM.